calcite-adapter/src/main/java/software/amazon/documentdb/jdbc/calcite/adapter/DocumentDbAggregate.java

```diff
@@ -166,7 +163,9 @@ public DocumentDbAggregate(final RelOptCluster cluster, final RelTraitSet traitS
                 "{$group: " + Util.toString(list, "{", ", ", "}") + "}");
         final List<String> fixups = getFixups(aggCalls, groupSet, inNames, outNames);
 
-        if (!groupSet.isEmpty() || aggCalls.stream().anyMatch(AggregateCall::isDistinct)) {
+        if (!groupSet.isEmpty()
+                || aggCalls.stream().anyMatch(AggregateCall::isDistinct)
+                || aggCalls.stream().anyMatch(aggregateCall -> aggregateCall.getAggregation() == SqlStdOperatorTable.SUM)) {
```

Comment:
Would this be more efficient if you OR'd the two condition inside one `anyMatch()` call?